lambda/src/personalizationUtil.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Personalization Utility provides person details and personalized prompts
+ **/
+ 'use strict';
+
+ const EMPTY = "";
+ 
+ /**
+  * 
+  * Get PersonalizedPrompt if skill personalization enabled else default
+  * @returns Personalized message if applicable
+  */
+ const getPersonalizedPrompt = (handlerInput) => {
+     if (getPerson(handlerInput)) {
+         return getPersonalizedPromptFromId(handlerInput);
+     }
+     return handleFallback;
+ }
+ /**
+  * Get person from requestEnvelope if personalization enabled.
+  * 
+  */
+ const getPerson = (handlerInput) => { return handlerInput.requestEnvelope.context.System.person };
+ 
+ /**
+  * Get person Id from requestEnvelope if personalization enabled.
+  */
+ const getPersonId = (handlerInput) => { return getPerson(handlerInput).personId };
+ 
+ /**
+  * Handle fallback logic incase person not found.
+  */
+ const handleFallback = () => { return EMPTY }
+ 
+ /**
+  * Get personalized greeting from person Id.
+  * Refer: https://developer.amazon.com/en-US/docs/alexa/custom-skills/add-personalized-greetings-or-prompts.html
+  */
+ const getPersonalizedPromptFromId = handlerInput => {
```

Comment:
should this be a function too 
```js
const getPersonalizedPromptFromId = (handlerInput) => { ...}

```

Reply:
Not necessarily , as this method is not exposed outside the Util and does not call any async API.